appveyor.yml

```diff
@@ -15,25 +15,19 @@ branches:
 
 # Install scripts. (runs after repo cloning)
 install:
-  - git rev-parse HEAD
-  # Get the latest stable version of Node 0.STABLE.latest
   - ps: Install-Product node $env:nodejs_version
-  # Install PhantomJS
-  - npm install phantomjs-prebuilt
-  - npm install mocha-appveyor-reporter
-  # Typical npm stuff.
-  - npm version
-  - npm install
+  - yarn global add phantomjs-prebuilt mocha-appveyor-reporter
+  - yarn install
 
 cache:
-  - '%APPDATA%\npm-cache'
+  - '%LOCALAPPDATA%\Yarn'
   - '%APPDATA%\Roaming\bower'
 
-
 # Post-install test scripts.
 test_script:
   # Output useful info for debugging.
-  - npm version
+  - git rev-parse HEAD
+  - yarn version
   - cmd: npm run test-all
```

Comment:
Can't use `yarn` to run for Node.js 0.12. This is a hybrid configuration right now, intentionally.